provider/pkg/gen/overrides.go

```diff
@@ -82,6 +87,13 @@ var resourceNameByPathOverrides = map[string]string{
 	"essentialcontacts_v1.json:v1/folders/{foldersId}/contacts":             "FolderContact",
 	"essentialcontacts_v1.json:v1/organizations/{organizationsId}/contacts": "OrganizationContact",
 
+	// GKEHub
+	"gkehub_v1alpha.json:v1alpha/projects/{projectsId}/locations/{locationsId}/memberships/{membershipsId}/rbacrolebindings": "",
```

Comment:
Why are these names blank? Does leaving it blank mean the resource gets skipped?